apps/tenant-management-webapp/src/app/store/configuration/sagas.ts

```diff
@@ -118,6 +104,29 @@ export function* fetchConfigurationDefinitions(_action: FetchConfigurationDefini
   }
 }
 
+async function fetchConfigurationNamespace(urlData: UrlData, token: string) {
+  let next = '';
+
+  const configurations = [];
+  let hasMoreData = true;
+
+  while (hasMoreData) {
+    let url = `${urlData.url}?top=10&after=${next}`;
+    const { data } = await axios.get(url, { headers: { Authorization: `Bearer ${token}` } });
+
+    configurations.push(...data.results);
+
+    if (data.page?.next) {
+      url = `/configuration/form-service?top=100&after=${data.page.next}`;
```

Comment:
Why is this hard coded to the form service?  All our services that have paging have the same interface, so why not use ${urlData.url}?top=...